### PR TITLE
Remove button

### PR DESCRIPTION
**Pull request by carlosahs**
Add remove button so that instructors can remove previous or duplicate courses.

How it looks: https://www.youtube.com/watch?v=FlMtntCSVy0
How it looks after applying the suggestions: https://www.youtube.com/watch?v=LPqOozkBkTc

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -41,6 +41,13 @@ def duplicate
     redirect_to @course.copy, flash: { success: t('courses.copied_succesfully') }
   end
 
+  def destroy
+    @course = Course.find(params[:id])
```
Comment (rhomeister) [inline app/controllers/courses_controller.rb:45]: the course will be automatically loaded by a library called `cancancan`, which is activated on line 4. Try if it still works after deleting this line.

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -41,6 +41,13 @@ def duplicate
...
+    @course.destroy
+
+    redirect_to action: 'index'
```
Comment (rhomeister) [inline app/controllers/courses_controller.rb:48]: To make it a bit more responsive to the user, you can add a flash:

```suggestion
    redirect_to action: 'index', flash: { success: t('courses.deleted_successfully', course.name) }
```
And add to Lin 32 in config/locales/en.yml something like:

`deleted_successfully: The course %{name} has been deleted successfully`

See https://guides.rubyonrails.org/action_controller_overview.html#the-flash

**Review by rhomeister — CHANGES_REQUESTED** [review]
Looking good! I added two very small suggestions. Nice work

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (carlosahs) [reply app/controllers/courses_controller.rb:45]: I tried, it works nice.

File: app/controllers/courses_controller.rb (hunk 2 above)
Comment (carlosahs) [reply app/controllers/courses_controller.rb:48]: It did not work with `flash: { success: t('courses.deleted_successfully', course.name) }`, so I used `flash[:notice] = "The course #{@course.name} has been deleted successfully" ` instead and did not modify `en.yml`. It appears to be working fine.

**Pull request merged**